okhttp-testing-support/src/main/java/okhttp3/OkHttpClientTestRule.kt

```diff
@@ -68,6 +73,7 @@ class OkHttpClientTestRule : TestRule {
   }
 
   @Synchronized private fun addEvent(event: String) {
+    logger.info(event)
```

Comment:
Is this going to make all our tests incredibly chatty?

Reply:
A bit, but also shows the MockWebserver is currently logging to STDERR in IDE.  Making this optional.  Good catch.